Gridgen to_disu6 errors

Test `t061_test_gridgen.py` should add a test for `.to_disu6(fname, writevertices)`. A quick attempt to use the module raises several errors:
```
>>> g.to_disu6('x.disu', True)
  File "flopy/utils/gridgen.py", line 1302, in to_disu6
    f.write('  NVERT {}\n'.format(gridprops['nvert']))
KeyError: 'nvert'
>>> g.to_disu6('x.disu', False)
Traceback (most recent call last):
  File "flopy/utils/gridgen.py", line 1321, in to_disu6
    a = gridprops[prop]

KeyError: 'ihc'
```
And further down in the source file, another error:
> ./flopy/utils/gridgen.py:1341:28: F821 undefined name 'nodes'

Tests for `.to_disv6(fname)` are also absent (and should also be added), but this method at least writes a file.

--- a/autotest/t061_test_gridgen.py
+++ b/autotest/t061_test_gridgen.py
@@ -114,6 +114,18 @@ def test_gridgen():
         mu = flopy.modflow.Modflow(version='mfusg', structured=False)
         disu = g.get_disu(mu)
 
+        # test writing a modflow 6 disu package
+        fname = os.path.join(cpth, 'mymf6model.disu')
+        g6.to_disu6(fname)
+        assert os.path.isfile(fname), \
+            'MF6 disu file not created: {}'.format(fname)
+
+        # test writing a modflow 6 disv package
+        fname = os.path.join(cpth, 'mymf6model.disv')
+        g6.to_disv6(fname)
+        assert os.path.isfile(fname), \
+            'MF6 disv file not created: {}'.format(fname)
+
     return
 
 

--- a/flopy/utils/gridgen.py
+++ b/flopy/utils/gridgen.py
@@ -1287,7 +1287,7 @@ class Gridgen(object):
 
         """
 
-        gridprops = self.get_gridprops()
+        gridprops = self.get_gridprops_disu6()
         f = open(fname, 'w')
 
         # opts
@@ -1338,7 +1338,7 @@ class Gridgen(object):
             # celldata -- not optimized for redundant vertices yet
             f.write('BEGIN CELL2D\n')
             iv = 1
-            for n in range(nodes):
+            for n in range(gridprops['nodes']):
                 xc, yc = self.get_center(n)
                 s = '  {} {} {} {} {} {} {} {}\n'.format(n + 1, xc, yc, 4, iv,
                                                          iv + 1, iv + 2,

